### ISSUE TITLE
[bug] Tauri 1.2: Keyboard shortcuts stopped working on macOS.

### ISSUE THREAD

**Issue by sprout2000** [tauri-apps/tauri#5589]
### Describe the bug

After updating Tauri to v1.2, keyboard shortcuts stopped working on macOS.

### Reproduction

```sh
npm create tauri-app
cd tauri-app
cargo tauri dev
```

### Platform and versions

```sh
% cargo tauri info

Environment
  › OS: Mac OS 13.0.0 X64
  › Node.js: 18.12.1
  › npm: 8.19.3
  › pnpm: 7.14.2
  › yarn: 1.22.19
  › rustup: 1.25.1
  › rustc: 1.65.0
  › cargo: 1.65.0
  › Rust toolchain: stable-aarch64-apple-darwin 

Packages
WARNING: no lock files found, defaulting to npm
  › @tauri-apps/cli [NPM]: 1.2.0
  › @tauri-apps/api [NPM]: Not installed!
  › tauri [RUST]: 1.2.0,
  › tauri-build [RUST]: 1.2.0,
  › tao [RUST]: 0.15.2,
  › wry [RUST]: 0.22.0,

App
  › build-type: bundle
  › CSP: unset
  › distDir: ../src
  › devPath: ../src
package.json not found

App directory structure
  ├─ src-tauri
  ├─ .vscode
  └─ src
```

**Comment by suous** [comment]
I am also reporting this issue😀:

The keyboard events of custom menu items such as `CustomMenuItem::new("preference", "Preference").accelerator("CmdOrCtrl+,")` and native menu items such as `MenuItem::CloseWindow` or `MenuItem::Quit` not working unless `MenuItem::EnterFullScreen` triggered through keyboard first.

- Tauri 1.1 menu keyboard accelerator works effectively.
- Manually clicking on the menu items works fine.
- Menu items controlled by `Command` (custom or native) not working (lost focus from the app when keyboard event triggered).
- Fullscreen accelerator controlled by `fn` works fine.
- Menu items controlled by `Command` becomes valid when `MenuItem::EnterFullScreen` is triggered through keyboard.
- Tested in both dev and release environments.
- Not tested on other operating systems.

**Comment by lucasfernog** [comment]
Duplicate of #5579. The fix is being deployed, you can see it here: https://github.com/tauri-apps/tao/pull/620

**Comment by lucasfernog** [comment]
Fixed in tao v0.15.3. Please run `cargo update` in the `src-tauri` folder.

**Comment by tw93** [comment]
when i run `npm run tauri build -- --target universal-apple-darwin`, the bug still exists.
but when i run `npm run tauri build -- --target x86_64-apple-darwin` and m1 is ok
and `npm run dev` is ok
